src/game/shell.c

```diff
@@ -187,6 +190,100 @@ void Shell_Wait(int nticks)
     }
 }
 
+void Shell_ValidateLevelTitle(char *str)
```

Comment:
The function is actually about sanitizing (mutating the input data), not validating (checking the input data for conformance).